docs/python_support/quickstart.md

```diff
@@ -0,0 +1,222 @@
+# Python support in OVMS - quickstart {#ovms_docs_python_support_quickstart}
+
+OpenVINO Model Server allows users to write custom processing nodes in Python, so they may have full control over what happens with the data reaching such node and what comes out of it.
+
+In this quickstart you'll create a servable with a single custom Python code that will expect a string and return the same string, but all in uppercase. 
+
+To acheive that let's follow the steps:
+1. Prepare Docker Image And Workspace
+2. Create Client Application
+3. Write Python Code For The Server 
+4. Prepare Graph Configuration File
+5. Prepare Server Configuration File
+6. Deploy OpenVINO Model Server
+7. Send Requests From The Client
+
+### Step 1: Prepare Docker Image And Workspace
+
+Let's have all the work done in a new `workspace` directory. Also let's  the following subdirectories:
+- `workspace/models` (catalog that will be mounted to the deployment)
+- `workspace/models/python_model` (catalog with Python servable specification)
+
+You can do that in one go:
+
+```bash
+mkdir -p workspace/models/python_model && cd workspace
+```
+
+Since changing all the letters in the string to uppercase is a very basic use case, the basic Python-enabled model server image without extra Python packages is sufficient. If you'd need some external modules, you would need specifically add them to the image. 
```

Comment:
```suggestion
Since changing all the letters in the string to uppercase is a very basic example, the basic Python-enabled model server image without extra Python packages is sufficient. If you need some external modules, add them to the image manually. 
```
Should we use shortcuts in our doc such as you'll/you'd etc? I think we avoided it up to this point to be more formal. I remember also the feedback from external team that we should avoid refering to the reader directly.